lisa/tools/cargo.py

```diff
@@ -117,6 +117,7 @@ def __install_dependencies(self) -> None:
 
     def build(
         self,
+        options: str = "",
```

Comment:
The options is flexible, but not easy to reuse logic. Please define specified arguments as much as possible.